File: 	1134.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
let alcool = 0;
let gasolina =0;
let diesel = 0;
let valor;

do{
    valor = lines.shift().split(" ");

    if(valor == 1){
    alcool++;
    }
    if(valor == 2){
        gasolina++;
    }
    if(valor == 3){
        diesel++;
    }
    
} while(valor!=4 )

console.log(`MUITO OBRIGADO`);
console.log(`Alcool: ${alcool}`);
console.log(`Gasolina: ${gasolina}`);
console.log(`Diesel: ${diesel}`);

File: 	1134.java
import java.util.Scanner; 
public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		int v[] = new int[3];
		
		while (n != 4) {
			
			if(n >= 1 && n <=3) {
				v[n-1]++;
		}
			n = sc.nextInt();
	}
		System.out.println("MUITO OBRIGADO");
        System.out.println("Alcool: " + v[0]);
        System.out.println("Gasolina: " + v[1]);
        System.out.println("Diesel: " + v[2]);
}
}
